source/tyr/tests/integration/conftest.py

```diff
@@ -167,14 +164,13 @@ def create_repositories_instance_env_variables():
     tmp_path = tempfile.mkdtemp(prefix='tyr_instance_auv_')
     source_directory = "{path}/ed/source".format(path=tmp_path)
     backup_directory = "{path}/ed/backup".format(path=tmp_path)
-    aliases_directory = "{path}/ed/aliases".format(path=tmp_path)
-    synonyms_directory = "{path}/ed/synonyms_file".format(path=tmp_path)
+    target_path = "{path}/ed/target_file".format(path=tmp_path)
 
     os.environ["TYR_INSTANCE_auv"] = (
         '{"instance":{"name":"auv","source-directory":"' + source_directory + '",'
-        '"backup-directory":"' + backup_directory + '","aliases_file":"' + aliases_directory + '",'
-        '"synonyms_file":"' + synonyms_directory + '","target-file":"ed/target_file",'
-        '"tmp_file":"/ed/tmp_file","is-free":true,"exchange":"exchange"},'
+        '"backup-directory":"' + backup_directory + '","target-file":"ed/target_file",'
+        '"tmp_file":"/ed/tmp_file","is-free":true,"exchange":"exchange",'
+        '"target-file":"' + target_path + '/data.nav.lz4"},'
```

Comment:
Seems to me that `target-file` is defined twice in this block.

Reply:
:+1: